google-http-client/src/main/java/com/google/api/client/json/webtoken/JsonWebToken.java

```diff
@@ -361,6 +362,12 @@ public String toString() {
     return Objects.toStringHelper(this).add("header", header).add("payload", payload).toString();
   }
 
+  public String getRawToken() {
```

Comment:
Since we're unlikely to get the API perfect on the first try, we should mark this @Beta to warn users it might change.